src/nunit.analyzers/ClassicModelAssertUsage/ContainsClassicModelAssertUsageCodeFix.cs

```diff
@@ -31,7 +31,7 @@ public sealed class ContainsClassicModelAssertUsageCodeFix
                     SyntaxFactory.SingletonSeparatedList(expectedArgument))));
 
             var actualArgument = argumentNamesToArguments[NUnitFrameworkConstants.NameOfActualParameter].WithNameColon(null);
-            return (actualArgument, constraintArgument);
+            return (actualArgument, constraintArgument.WithTriviaFrom(actualArgument));
```

Comment:
@manfred-brands I think this is where I'd need your help here. I had tried to move it to `ClassicModelAssertUsageCodeFix`, but it appeared to me that there is no one simple solution that covers all assertions.

- If you see https://github.com/nunit/nunit.analyzers/blob/master/src/nunit.analyzers/ClassicModelAssertUsage/ClassicModelAssertUsageCodeFix.cs#L60-L68, we get a _new_ `actual` argument and a _new_ `constraint` argument from `ConstructActualAndConstraintArguments`, which is a `virtual` method of `ClassicModelAssertUsageCodeFix`.
- However, within the implementation of `ConstructActualAndConstraintArguments` in the subclasses of `ClassicModelAssertUsageCodeFix` (like this class), this `actualArgument` before the `return` is the _old_ argument.
- `ClassicModelAssertUsageCodeFix` does have the mapping `argumentNamesToArguments`, but it doesn't know the where the trivia of the new arguments should come from.
  - For most assertions, the parameter names are `expected` and `actual`.
  - For `ClassicAssert.Greater`, the parameter names are `arg1` and `arg2`.
  - For `ClassicAssert.IsTrue`, the parameter name is `condition`.
- Therefore, it appears that handling trivia in the individual codefixes would be the only way, but then it would be easy to miss/forget when adding a new one. As such, I wasn't entirely sure where the logic should live.
  - Or... maybe it's okay because there won't be any more `ClassicModelAssertUsageCodeFix`es.

**Given the above, do you think the trivia handling logic should live in `ClassicModelAssertUsageCodeFix` or its implementation?**

Reply:
Edit: I think I got this.